frontend/src/components/code-editor/helpers.ts

```
@@ -0,0 +1,78 @@
+          updatedSet = set.update({ add: [breakpointMarker.range(e.value.pos)] });
+        } else if (!e.value.on) {
+          updatedSet = set.update({ filter: (from) => from !== e.value.pos });
+        }
+      }
+    }
+    return updatedSet;
+  },
+});
+
+function toggleBreakpoint(view: EditorView, pos: number) {
+  let breakpoints = view.state.field(breakpointState);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'breakpoints' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)